components/studio/bin/hab-studio.sh

```diff
@@ -774,48 +774,7 @@ rm_studio() {
 
   info "Destroying Studio at $HAB_STUDIO_ROOT ($STUDIO_TYPE)"
 
-  # Set the verbose flag (i.e. `-v`) for any coreutils-like commands if verbose
-  # mode was requested
-  if [ -n "$VERBOSE" ]; then
-    local v="-v"
-  else
-    local v=
-  fi
-
-  # Unmount filesystems that were previously set up in, but only if they are
-  # currently mounted. You know, so you can run this all day long, like, for
-  # fun and stuff.
-
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/src type"; then
-    $bb umount $v -l $HAB_STUDIO_ROOT/src
-  fi
-
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/run type"; then
-    $bb umount $v $HAB_STUDIO_ROOT/run
-  fi
-
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/sys type"; then
-    $bb umount $v $HAB_STUDIO_ROOT/sys
-  fi
-
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/proc type"; then
-    $bb umount $v $HAB_STUDIO_ROOT/proc
-  fi
-
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/dev/pts type"; then
-    $bb umount $v $HAB_STUDIO_ROOT/dev/pts
-  fi
-
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/dev type"; then
-    $bb umount $v -l $HAB_STUDIO_ROOT/dev
-  fi
-
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/var/run/docker.sock type"; then
-    $bb umount $v -l $HAB_STUDIO_ROOT/var/run/docker.sock
-  fi
-
-  # Remove remaining filesystem
-  $bb rm -rf $v $HAB_STUDIO_ROOT
```

Comment:
We'll want to keep the deletion of remaining Studio filesystem in the `hab-studio rm` subcommand and not in the generic cleanup as the current behavior of the Studio is to preserve your work between running `new`, `enter`, and `run` (`build` is the only subcommand which calls `hab-studio rm` at the beginning of its run).